chain-impl-mockchain/src/transaction/payload.rs

```diff
@@ -91,8 +91,8 @@ impl<'a, P: Payload> PayloadAuthSlice<'a, P> {
 }
 
 impl<'a, P: Payload> PayloadSlice<'a, P> {
-    pub fn into_certificate_slice(self) -> Option<CertificateSlice<'a>> {
-        <P as Payload>::to_certificate_slice(self)
+    pub fn inpayload_to_certificate_slice(self) -> Option<CertificateSlice<'a>> {
```

Comment:
Is this intentional or caused by some search and replace :)? I would also keep the `into_certificate_slice` name as it is here

Reply:
Yes, my bad :D